scripts/globals/mobskills/throat_stab.lua

```diff
@@ -19,6 +19,10 @@ mobskill_object.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    if mob:getFamily() == 271 then -- Jailer of Love, uses only animation.
```

Comment:
missed one

Reply:
Also JoL can throat stab??? Thats legit in a capture???

Reply:
Yup lol

![image](https://user-images.githubusercontent.com/6257062/185818013-1a483b4a-79b0-43cd-81fb-8547f6362970.png)

Reply:
its one of those weird: just an animation abilities

Reply:
And last one should be addressed

Reply:
for me, it's (Vulture 4), best tp move

Reply:
> its one of those weird: just an animation abilities

as in its melee is mobskills or as in it literally tries to animate doing a skill that does no message/damage?

Reply:
Its just a 2hr animation

Reply:
Is that capture cut off? can you upload the whole thing?

there's probably a better way than to force use a TP move, going to look into it